src/app/store/effects/VehicleTestResultModel.effects.spec.ts

```diff
@@ -0,0 +1,91 @@
+import {Observable, of, ReplaySubject} from 'rxjs';
+import {TechnicalRecordServiceMock} from '../../../../testconfig/services-mocks/technical-record-service.mock';
+import {TestBed} from '@angular/core/testing';
+import {TechnicalRecordService} from '@app/technical-record-search/technical-record.service';
+import {RouterTestingModule} from '@angular/router/testing';
+import {INITIAL_STATE, Store} from '@ngrx/store';
+import {hot} from 'jasmine-marbles';
+import {provideMockActions} from '@ngrx/effects/testing';
+import {EVehicleTestResultModelActions,
+        GetVehicleTestResultModel,
+        GetVehicleTestResultModelFailure,
+        GetVehicleTestResultModelSuccess} from '@app/store/actions/VehicleTestResultModel.actions';
+import {IAppState} from '@app/store/state/app.state';
+import {VehicleTestResultModelEffects} from '@app/store/effects/VehicleTestResultModel.effects';
+import {TestResultService} from '@app/technical-record-search/test-result.service';
+import {TestResultModel} from '@app/models/test-result.model';
+
+const testResult: TestResultModel = {
```

Comment:
For testing we need this much data. Please lets discuss the approach

Reply:
I added more data because the error on the tests failing specifically said that data for field x from TestResultModel was missing. Sure, we can discuss any approach.

Reply:
yes there is a simpler approach. for example `const testResult = {} as TestReult`. will work just fine. You only need to show the intentions here and not the either payload

Reply:
show few keys with that approach.